src/azul/chalice.py

```diff
@@ -13,9 +14,12 @@ def route(self, path, **kwargs):
         Same as method in supper class but stashes URL path a view function is bound to as an attribute of the
         function itself.
         """
+        description = kwargs.pop('spec', None)
```

Comment:
```suggestion
        spec = kwargs.pop('spec', None)
```

Reply:
This will cause a name collision. I will import the spec decorator so that it's use is qualified (to avoid the collision).